content/4_Gold/Conclusion.mdx

```diff
@@ -0,0 +1,56 @@
+---
+id: gold-conclusion
+title: Additional Practice for USACO Gold
+author: Nathan Wang
+description:
+  'Congratulations on making it this far! Here are some final tips for Gold, as
+  well as additional practice problems you may find useful.'
+---
+
+## What Now?
+
+After completing this section on the USACO Guide, you should be comfortable with
+the topics that have historically appeared in the USACO Gold
+division.<Asterisk>Of course, there's always a chance that another topic,
+potentially from a higher division, will show up in future Gold
+contests.</Asterisk> In order to maximize your chance at promoting, you should
+aim to **practice** by doing more problems.
+
+## Uncategorized Problems
+
+Below we list some problems that we think are roughly Gold level but don't
+really fit in previous modules.<Asterisk>either due to too many problems or
+because it requires more than one concept</Asterisk> The difficulties listed
+here are relative to the Gold division (difficulty is subjective and isn't
+always accurate though).
+
+// todo: massive problems list
+
+## Where to find more problems
+
+1. Old USACO problems -- We provide a list of recent USACO problems
+   [here](https://usaco.guide/general/usaco-monthlies). You can also find even
+   older USACO problems on the [USACO](http://www.usaco.org/) website. Note that
+   older USACO problems may be easier than recent USACO contest problems due to
+   increases in difficulty.
+2. [Codeforces](https://codeforces.com/) -- you can search by tag, difficulty
+   level, etc. For a _rough_ guideline of CodeForces rating to USACO division
+   mapping, check out
+   [our FAQ page](https://usaco.guide/usaco-faq/#q-what-codeforces-rating-corresponds-to-each-of-the-usaco-divisions).
+3. [CSAcademy Archive](https://csacademy.com/contest/archive/)
+
+## Final Thoughts
+
+If you find yourself getting demotivated because you can't solve problems by
+yourself, consider changing your goal to "how many problems can I solve per
```

Comment:
This demotivates people?! I get annoyed problems are too easy if I solve them w/o editorial!

Reply:
legend 😍 we can add a "on the flip side, if you're solving too many problems w/o editorial, do harder problems" or something idk :P

Reply:
yea possibly